src/Miunie.Discord/CommandHandler.cs

```diff
@@ -53,6 +54,9 @@ public async Task InitializeAsync()
             _ = await _commandService.AddModulesAsync(Assembly.GetExecutingAssembly(), _services);
         }
 
+        public HelpCommandProvider GetHelpProvider(ILanguageProvider lang)
+            => new HelpCommandProvider(_commandService, lang);
+
```

Comment:
Would it be fine to initialize the `CommandService` in the IoC? The reason I made this method was because the `CommandService` was initialized in `CommandHandler`.

Reply:
The `CommandService` should definitely be injected as well.
I think there was some issue with it not resolving properly, but we need it to be properly injected.